modules/pytket-honeywell/pytket/extensions/honeywell/backends/honeywell.py

```diff
@@ -532,14 +545,14 @@ def save_login(user_name: str, pwd: str) -> None:
         """
         set_honeywell_config(user_name)
 
-        keyring.set_password(HoneywellQAPI.KEYRING_NAME, user_name, pwd)  # type: ignore
+        keyring.set_password(PersistentStorage.KEYRING_SERVICE, user_name, pwd)  # type: ignore
 
     @staticmethod
     def clear_saved_login(user_name: str) -> None:
         """Delete saved password for user_name if it exists"""
         set_honeywell_config(None)
         try:
-            keyring.delete_password(HoneywellQAPI.KEYRING_NAME, user_name)  # type: ignore
+            keyring.delete_password(PersistentStorage.KEYRING_SERVICE, user_name)  # type: ignore
```

Comment:
These are useful changes but they don't appear to be related to batching. Could these be in a separate PR (or a separate commit if they make sense to be grouped with the batching changes)?

Reply:
Separate commit makes sense - I'm happy with this change.